features/machine/machine_healthcheck.feature

```diff
@@ -72,3 +72,51 @@ Feature: MachineHealthCheck Test Scenarios
     When I create the 'Ready' unhealthyCondition
 
     Then the machine should be remediated
+
+  # @author jhou@redhat.com
+  # @case_id OCP-25691
+  @admin
+  @destructive
+  Scenario: Use "maxUnhealthy" to prevent automated remediation
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+
+    Given I store the number of machines in the :num_to_restore clipboard
+    And admin ensures node number is restored to "<%= cb.num_to_restore %>" after scenario
```

Comment:
@jhou  , Here do we have to take care of number of nodes , as after the cloned  machineset is deleted , will that also be taken care by itself ?

Reply:
Yes, we must take care of the number of nodes, wait until it's deleted to exit. This is to ensure it  does not break the next test following it.